README.md

```diff
@@ -16,7 +16,9 @@ Alternatively, press `Ctrl`+`Space` (Windows, Linux) or `Cmd`+`Space` (macOS) to
 | ---------------------------- | -------------------------------------------------------------------- |
 | `spring-crudctrl`            | Spring controller with CRUD actions implemented                      |
 | `spring-crudrepo`            | Spring Data JPA CRUD repository with REST api annotation             |
+| `spring-jparepo`            | Spring Data JPA repository with REST api annotation             |
 | `spring-entity`              | JPA entity with ID field                                             |
+| `spring-lombok-entity`              | JPA entity with ID field and Lombok `@Data` annotation                                            |
```

Comment:
Snippets description To either use Lombok or not and JPA repository or CRUD repository.